jcache/remote/src/main/java/org/infinispan/jcache/remote/JCache.java

```diff
@@ -428,61 +428,6 @@ protected Object getCacheStatisticsMXBean() {
       return this;
    }
 
-   private JCache<K, V> checkNotNullOrNullKV(Map<? extends K, ? extends V> map, String name) {
-      checkNotNull((Object) map, name);
-      if (containsNullKey(map)) {
-         throw log.parameterMustNotContainNullKeys(name);
```

Comment:
I wonder if we need internationalization for this still, cause unfortunately we lose that with the new approach.  @galderz can you comment on that?